test/Microsoft.AspNetCore.Mvc.Core.Test/ModelBinding/ParameterBinderTest.cs

```diff
@@ -836,6 +836,60 @@ public static TheoryData BindModelAsyncData
                 });
         }
 
+        // Regression test for aspnet/Mvc#8078. Later parameter should not mark entry as valid.
+        [Fact]
+        public async Task BindModelAsync_ForOverlappingParameters_InValid_WithInValidFirstParameterAndSecondNull()
```

Comment:
Ran this test without `src` change. It failed as expected 😺 at [line 87](https://github.com/aspnet/Mvc/pull/8154/files#diff-6c9db3c93c535f2f6e15b7ad5a471648R887).